shared/protos/garbage/garbage.proto

```diff
@@ -0,0 +1,234 @@
+syntax = "proto3";
+
+package google.garbage.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/resource.proto";
+import "google/longrunning/operations.proto";
+import "google/protobuf/timestamp.proto";
+
+option (google.api.client_package) = {
+  title: "Testing Garbage"
+  namespace: ["Google", "Cloud"]
+  version: "v1"
+};
+
+// Endless trash
+service GarbageService {
+  // The address where this API is typically served.
+  option (google.api.default_host) = "garbage.googleapis.com";
```

Comment:
Same. More random trash and less Google references.

Reply:
Trashed it up.